circle.yml

```diff
@@ -44,6 +44,8 @@ machine:
   environment:
     MYSQL_USER: ubuntu
     MYSQL_DB: circle_test
+    # Quiet Maven invoker logs (Downloading... when running zipkin-server/src/it)
+    MAVEN_OPTS: -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
might need to add java heap size args.. not sure why the build died from out-of-memory (haven't looked to see if it was doing this formerly)